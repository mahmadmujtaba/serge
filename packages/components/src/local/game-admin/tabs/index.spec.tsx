/* global it expect */

import React from 'react'
import renderer from 'react-test-renderer'

import Tabs from './index'

it('tabs renders correctly', () => {
  const tree = renderer
    .create(<Tabs tabs={['tab1', 'tab2', 'tab3']} disabled={false}>Text</Tabs>)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
