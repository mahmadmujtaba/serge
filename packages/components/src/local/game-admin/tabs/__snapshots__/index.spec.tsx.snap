// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tabs renders correctly 1`] = `
<div
  className="main"
>
  <div
    className="item active"
    onClick={[Function]}
  >
    tab1
  </div>
  <div
    className="item"
    onClick={[Function]}
  >
    tab2
  </div>
  <div
    className="item"
    onClick={[Function]}
  >
    tab3
  </div>
</div>
`;
